src/clojurewerkz/elastisch/native/conversion.clj

```diff
@@ -1271,6 +1271,10 @@
           {}
           (.getIndices r)))
 
+(defn ^IPersistentMap indices-stats-response->map
+  "transforms indices stats response to the Clojure hash-map"
+  [^IndicesStatsResponse r]
+  (json/parse-string (.toString r) true))
```

Comment:
This works and I assume efficiency in this particular case won't matter but a more common native client way of doing it would be to use `IndicesStatsResponse#asMap` and convert the result without serialising to and parsing JSON.